tests/unit/lms/models/public_id_test.py

```diff
@@ -10,9 +10,7 @@
 class TestPublicId:
     def test_it_generates_ids(self):
         public_id = PublicId(
-            region=Region(
-                code="us", name="Worldwide (U.S.)", authority="lms.hypothes.is"
-            ),
+            region=Region(code="us", authority="lms.hypothes.is"),
```

Comment:
So now anywhere that constructs a `Region` benefits from not having to pass in `name` themselves. Much nicer.